packages/wrangler/src/config/index.ts

```diff
@@ -130,6 +134,31 @@ export function findWranglerToml(
 	return findUpSync(`wrangler.toml`, { cwd: referencePath });
 }
 
+/**
+ * Pages projects require the `pages_build_output_dir` to be specififed in
+ * their `wrangler.toml` file. This function returns the configuration file
+ * path, only if the configuration is valid for Pages.
+ */
+export function findPagesWranglerToml(
```

Comment:
let's discuss these two in the context of https://github.com/cloudflare/workers-sdk/pull/5449

Reply:
Could you adapt this to use #5449?

Reply:
done